src/main/java/com/minecolonies/blockout/controls/TextField.java

```diff
@@ -367,27 +367,27 @@ else if (cursorBeforeEnd && shadow)
         if (visibleString.length() > 0 && cursorVisible && relativeCursorPosition < visibleString.length())
         {
             mc.renderEngine.bindTexture(TEXTURE);
-            mc.fontRenderer.drawString(visibleString.substring(relativeCursorPosition), textX, drawY, color, shadow);
+            mc.fontRendererObj.drawString(visibleString.substring(relativeCursorPosition), textX, drawY, color, shadow);
         }
 
         //  Should we draw the cursor this frame?
         if (isFocus() && cursorVisible && (cursorBlinkCounter / 6 % 2 == 0))
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Assign this magic number 6 to a well-named constant, and use the constant instead. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonar.minecolonies.com/coding_rules#rule_key=squid%3AS109)